appservice/src/getKuduClient.ts

```diff
@@ -8,10 +8,15 @@ import { BasicAuthenticationCredentials } from 'ms-rest';
 import { addExtensionUserAgent } from 'vscode-azureextensionui';
 import KuduClient from 'vscode-azurekudu';
 import { ArgumentError } from './errors';
+import { localize } from './localize';
 import { SiteClient } from './SiteClient';
 
 export async function getKuduClient(client: SiteClient): Promise<KuduClient> {
     const user: User = await client.getWebAppPublishCredential();
+    if (client.isLinuxConsumptionPlan) {
+        const notSupportedLinux: string = localize('notSupportedLinux', 'This action is not supported with Linux Consumption Plans.');
```

Comment:
I don't want to base _whether we throw an error_ off of the plan type. That should be based off of `if (!this.kuduUrl)` or something like that. Specifically, this logic wouldn't handle these two cases:
1. The app service team adds support for kudu down the line on linux consumption
1. They add another plan type that _doesn't_ support kudu.

Maybe the message could be something like this where you just insert site.kind and plan.sku.tier:
```
`This operation is not supported by App Service Plans with kind "{...}" and sku "{...}"`
```